tokio-postgres/src/config.rs

```diff
@@ -56,9 +56,12 @@ pub enum ChannelBinding {
     Require,
 }
 
+/// A host specification.
 #[derive(Debug, Clone, PartialEq)]
-pub(crate) enum Host {
+pub enum Host {
+    /// A TCP hostname.
     Tcp(String),
+    /// A path to a directory containing the server's Unix socket.
```

Comment:
Can you add a note that this variant is only available on Unix platforms?

Reply:
Yep, done!